moz_sql_parser/sql_parser.py

```diff
@@ -371,9 +375,71 @@ def to_string(instring, tokensStart, retTokens):
 ).addParseAction(to_union_call)
 
 
-SQLParser = selectStmt
+def to_create_json_call(instring, tokensStart, retTokens):
+    # ARRANGE INTO {op: params} FORMAT
+    tok = retTokens
+    name = tok[0][0].lower()
+
+    types = tok.types
+    if not types:
+        types = tok[0][1]
+    return {"name" : name, "type": types}
+
+
+
+def to_create_table_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    if tok:
+        return {"create table" : tok.asList()}
+
+def to_table_name_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    if tok[0]:
+        return {"name" : tok[0] }
+
+
+def to_columns_call(instring, tokensStart, retTokens):
+    tok = retTokens
+
+    return {"columns" : tok.asList()}
+
+
+createStmt = Forward()
+
+column_name = ident.copy().setName("column_name").setDebugActions(*debug)
+
+column_size = Group(
+                Literal('(').suppress().setDebugActions(*debug) +
+                Word(nums) +
```

Comment:
` Word(nums)` is not correct; use `intNum` pattern

Reply:
In intNum, we consider negative numbers also. So our grammar can accept negative numbers. That's why I choose only `nums`

Reply:
Done!